plutus-core/plutus-core/src/PlutusCore/Error.hs

```diff
@@ -67,6 +67,9 @@ data ParseError ann
     | InvalidBuiltinConstant ann T.Text T.Text
     deriving (Eq, Generic, NFData, Functor)
 
+instance (Eq ann) => (Ord (ParseError ann)) where
+    compare _ _ = EQ
```

Comment:
Could we derive it just like `Eq`? Or not have it all? Or have a comment explaining the weird instance?

Reply:
Added comment.

Reply:
I'd still derive it, who knows what they use it for.

Reply:
I derived it at first, but then I need to derive `Ord` instances for a few more things - but I guess that's not a problem. (It was causing some other problems before but there seems to be no problem now.)

Reply:
Pushed.

Reply:
:+1: